pages/12.Plugins/08.Hubspot/docs.en.md

```diff
@@ -44,6 +44,9 @@ It is checked by default. If you uncheck it, the plugin will not push contacts t
 
 1. Set up the [cron job][cron-job] if you have not already configured it.
 
+1. Script to configure in your cron job: `php $PATH_TO_MAUTIC_DIRECTORY/bin/console mautic:integration:fetchleads --integration=Hubspot --fetch-all`
```

Comment:
I would recommend that we display this in an info box:

```suggestion
>>> Script to configure in your cron job: `php $PATH_TO_MAUTIC_DIRECTORY/bin/console mautic:integration:fetchleads --integration=Hubspot --fetch-all`
```

Reply:
Nice formatting suggestion. Just applied it 🤓